src/main/java/org/apache/sling/resourceresolver/impl/mapping/AliasHandler.java

```diff
@@ -130,31 +132,30 @@ protected boolean initializeAliases() {
                     } else if (!conflictingAliases.isEmpty()) {
                         log.warn("There are {} conflicting aliases: {}", conflictingAliases.size(), conflictingAliases);
                     }
+
                     if (invalidAliases.size() >= MAX_REPORT_DEFUNCT_ALIASES) {
                         log.warn("There are {} invalid aliases; excerpt: {}", invalidAliases.size(), invalidAliases);
                     } else if (!invalidAliases.isEmpty()) {
                         log.warn("There are {} invalid aliases: {}", invalidAliases.size(), invalidAliases);
                     }
                 } catch (final Exception e) {
-
+                    this.aliasMapsMap = new ConcurrentHashMap<>();
                     logDisableAliasOptimization(e);
-
-                    // disable optimize alias resolution
-                    isOptimizeAliasResolutionEnabled = false;
                 }
             }
 
             doUpdateConfiguration.run();
             sendChangeEvent.run();
-
-            return isOptimizeAliasResolutionEnabled;
-
         } finally {
 
             this.initializing.unlock();
         }
     }
 
+    boolean usesCache() {
```

Comment:
I wonder if this method name is correct, as it returns the value of the variable ``mapsIsInitialized``... (or maybe the name of the variable should be improved?)

Reply:
better 👍